integrationTests/chainSimulator/bridge/executeOrDepositFail_test.go

```diff
@@ -371,3 +371,94 @@ func getTokenMetaData(token chainSim.ArgsDepositToken) *esdt.MetaData {
 		Creator: bytes.Repeat([]byte{1}, 32),
 	}
 }
+
+func TestChainSimulator_ExecuteSameNft2Times(t *testing.T) {
```

Comment:
Weird test for creating same NFT 2 times in the same cross-chain transfer.
- execute operation with 2 NFTs (same token in both)
- will try to create NFT 2 times, second create will override quantity because has same key
- will try to send 2 NFTs, but 2nd send will fail because quantity was 1
- in the end, the transaction is failed and 1 NFT will stay in mainchain, 1 will be gone